tools/depends/target/Makefile

```diff
@@ -6,13 +6,13 @@ endif
 
 DEPENDS = \
 	pcre expat gettext sqlite3 libgpg-error \
-	libgcrypt bzip2 libfstrcmp liblzo2 freetype2 fontconfig \
+	libgcrypt bzip2 libfstrcmp liblzo2 freetype2-nohb fontconfig \
```

Comment:
I would name this `freetype2-noharfbuzz` for clarity. I knew what it meant, but still had to think about "hb" for a sec the first time I saw it in the code outside this PR.